.github/workflows/test_thorough.bat

```diff
@@ -0,0 +1,6 @@
+set grass=%1\grass79
+set python=C:\OSGeo4W64\bin\python3
+
+call %grass% --tmp-location XY --exec g.extension g.download.location
+call %grass% --tmp-location XY --exec g.download.location url=http://fatra.cnr.ncsu.edu/data/nc_spm_full_v2alpha2.tar.gz dbase=%USERPROFILE%
+call %grass% --tmp-location XY --exec %python% -m grass.gunittest.main --grassdata %USERPROFILE% --location nc_spm_full_v2alpha2 --location-type nc --min-success 60
```

Comment:
It seems that having equivalent of --exec which would call directly whatever Python is GRASS GIS using. If you want you can create an issue for it, but we can just proceed with the PR without it.

Reply:
Something like `--python` would be great, so we don't have to worry about finding the right python executable.

Reply:
Would you mind opening a ticket for it?

Reply:
Sure. #703

Reply:
Thanks!